src/main/types/workspaces/Workspace.js

```diff
@@ -121,15 +120,26 @@ class Workspace {
   }
 
   delete() {
+    let workspaceDirectory;
+    if (fse.lstatSync(this.workspaceDirectory).isSymbolicLink()) {
+      workspaceDirectory = fse.readlinkSync(this.workspaceDirectory);
+      fse.unlinkSync(this.workspaceDirectory);
+    } else {
+      workspaceDirectory = this.workspaceDirectory;
+    }
+
     try {
-      fse.removeSync(this.workspaceDirectory);
+      fse.removeSync(workspaceDirectory);
     } catch (e) {
```

Comment:
At first glance this worried me, because we unlink _and_ delete, but we're deleting the symlink, _and_ the directory to which that symlink points.